### PR TITLE
Adding a Mod Menu to your GDPS

### PR DESCRIPTION
**Pull request by Priestytheplushie**
Guide on adding a mod menu to your GDPS

### CODE REVIEW

File: in-game-help/add-mod-menu.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+15. Copy the result and find and replace "aHR0cDovL3d3dy5ib29tbGluZ3MuY29tL2RhdGFiYXNl" with the GDPS's encoded link
+16. Save the File
+17. Open APK Editor
+18. Click on the APK of the **Mod Menu**
+19. Click "Simple Edit"
+20. Click on "lib"
+21. Click on "armebi-v7a"
+22. Replace the "libcocos2dcpp.so" with the modified one you created
+23. Click "Build"
+
```
Comment (Redactedslair) [inline in-game-help/add-mod-menu.md:40]: Not the biggest problem, but there are no images to preview what you should do which would be nice to have.

**Comment by Redactedslair** [comment]
apart from the no images part, it seems good!

**Comment by Priestytheplushie** [comment]
ill add some images

File: in-game-help/add-mod-menu.md (hunk 1 above)
Comment (Priestytheplushie) [reply in-game-help/add-mod-menu.md:40]: on it

**Comment by xavwashere** [comment]
Overall, this is very useful. Only thing I might ask though, this seems to be Android only. You could make a tutorial on how to add, say, Mega Hack v7/v5 to your GDPS. Just a thought!

**Comment by Priestytheplushie** [comment]
I tried to explain adding GDMegaOverlay which is a free alternetive to Mega Hack, as not everyone has it

**Comment by Redactedslair** [comment]
> Overall, this is very useful. Only thing I might ask though, this seems to be Android only. You could make a tutorial on how to add, say, Mega Hack v7/v5 to your GDPS. Just a thought!

with megahack v6-7 its very simple to install it and megahack v5 should autodetect if there is a geometry dash instance running (any version or gdps)

**Comment by Redactedslair** [comment]
> I tried to explain adding GDMegaOverlay which is a free alternetive to Mega Hack, as not everyone has it

the only thing left is to add it to SUMMARY.md so people could see it.

**Comment by Priestytheplushie** [comment]
how do i do that lol

**Comment by Priestytheplushie** [comment]
ok i fixed it

**Review by Redactedslair — APPROVED** [review]
Seems good now. 👍

**Comment by Priestytheplushie** [comment]
can you merge my pull request?

**Comment by Redactedslair** [comment]
> can you merge my pull request?

sadly no, since i do not have write access which is required to merge it.

**Comment by xavwashere** [comment]
> > Overall, this is very useful. Only thing I might ask though, this seems to be Android only. You could make a tutorial on how to add, say, Mega Hack v7/v5 to your GDPS. Just a thought!
> 
> with megahack v6-7 its very simple to install it and megahack v5 should autodetect if there is a geometry dash instance running (any version or gdps)

In Mega Hack v5, it won't work without this specific setup:
1. Your GDPS .exe file needs to be named "GeometryDash.exe"
2. The save data folder (AppData/Local/{your gdps's name}) needs to be renamed to "GeometryDash". This can create conflicts between the GDPS and your main GD save.

It should then work.

As for Mega Hack v7, installing it worked, but upon opening the game, it would immediately crash.

**Comment by Priestytheplushie** [comment]
> > > Overall, this is very useful. Only thing I might ask though, this seems to be Android only. You could make a tutorial on how to add, say, Mega Hack v7/v5 to your GDPS. Just a thought!
> > 
> > 
> > with megahack v6-7 its very simple to install it and megahack v5 should autodetect if there is a geometry dash instance running (any version or gdps)
> 
> In Mega Hack v5, it won't work without this specific setup:
> 
> 1. Your GDPS .exe file needs to be named "GeometryDash.exe"
> 2. The save data folder (AppData/Local/{your gdps's name}) needs to be renamed to "GeometryDash". This can create conflicts between the GDPS and your main GD save.
> 
> It should then work.
> 
> As for Mega Hack v7, installing it worked, but upon opening the game, it would immediately crash.

than just use GDMegaOverlay 🤷

**Comment by Redactedslair** [comment]
> GDPS. 

Mhv7 crashing is a you problem. (It doesn't crash for me).
As for mhv5 its valid, but of course you can always modify the executable file to make it work.

**Comment by Jecket22** [comment]
This would benefit a bit from better writing, but works well enough for now.

**Pull request merged**